beacon-chain/state/stategen/service.go

```diff
@@ -96,6 +127,9 @@ func (s *State) Resume(ctx context.Context, fState state.BeaconState) (state.Bea
 	fRoot := bytesutil.ToBytes32(c.Root)
 	// Resume as genesis state if last finalized root is zero hashes.
 	if fRoot == params.BeaconConfig().ZeroHash {
+		if s.MinimumSlot() > 0 {
+			return nil, fmt.Errorf("no finalized checkpoint, and history before slot %d not currently available", s.MinimumSlot())
```

Comment:
`currently` implies that it's a temporary thing, but since the database was weak subjectively synced, the history will never be available, right? If yes, then I would remove the word `currently`.

Reply:
I was thinking of backfill here,  but as we do not currently have backfill it would be reasonable to word this differently.

Reply:
removed "currently"